shell/platform/darwin/ios/framework/Source/FlutterDartProject_Internal.h

```diff
@@ -8,11 +8,12 @@
 #include "flutter/common/settings.h"
 #include "flutter/runtime/platform_data.h"
 #include "flutter/shell/common/engine.h"
-#import "flutter/shell/platform/darwin/ios/framework/Headers/FlutterDartProject.h"
+#import "flutter/shell/platform/darwin/common/framework/Headers/FlutterDartProject.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 NSBundle* FLTFrameworkBundleInternal(NSString* bundleID, NSURL* searchURL);
+NSBundle* FLTFrameworkBundleWithIdentifier(NSString* bundleID);
```

Comment:
These should be declared in a `common/framework/source/FlutterDartProjectHelper.h`, not twice in two different headers.